`4.18` Release Figma Changelog (#2806)
hashicorp/design-system: packages/components/CHANGELOG-FIGMA-COMPONENTS.md

# [HDS Components UI Kit v2.0](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&node-id=2-7&t=HYGTIoXBy2YkVWDP-1)

> ### April 2nd, 2025

Reviewer: ```suggestion
## April 2nd, 2025
```

`Stepper List` and `Stepper Nav` - Components added.

`Advanced Table` - Multiple enhancements including:

- Added support for expanding and collapsing all rows.
- Added support for a sticky column in the [Template] components.

`Code Block` and `Code Editor` - Improved the color contrast in the focus ring.

> ### March 5th, 2025

Reviewer: ```suggestion
## March 5th, 2025
```

Author: Whoops, good catch!

# [HDS Components UI Kit v2.0](https://www.figma.com/design/iweq3r2Pi8xiJfD9e6lOhF/HDS-Components-v2.0?m=auto&node-id=2-7&t=HYGTIoXBy2YkVWDP-1)

## April 2nd, 2025

> `Stepper List` and `Stepper Nav` - Components added.

Reviewer 2: ```suggestion
`StepperList` and `StepperNav` - Components added.
```

> `Advanced Table` - Multiple enhancements including:

Reviewer 2: ```suggestion
`AdvancedTable` - Multiple enhancements including:
```

- Added support for expanding and collapsing all rows.
- Added support for a sticky column in the [Template] components.

> `Code Block` and `Code Editor` - Improved the color contrast in the focus ring.

Reviewer 2: ```suggestion
`CodeBlock` and `CodeEditor` - Improved the color contrast in the focus ring.
```
